.travis.yml

```diff
@@ -1,18 +1,30 @@
 language: python
 
 python:
-  - 3.6
+- 3.6
 
 env:
-  - TOX_ENV=python
-  - TOX_ENV=flake8
+- TOX_ENV=python
+- TOX_ENV=flake8
+- TOX_ENV=package
 
 install:
-  - pip install --upgrade pip
-  - pip install tox coveralls
+- pip install --upgrade pip
+- pip install tox coveralls
 
 script:
-  - tox -e $TOX_ENV
+- tox -e $TOX_ENV
 
 after_success:
-  - test $TOX_ENV = "python" && coveralls
+- test $TOX_ENV = "python" && coveralls
+
+deploy:
```

Comment:
This section allows Travis to take the eplaunch.tar.gz file and push it to Github.  There are lots of nuances here:
 - the build cleanup step must be skipped
 - the API key is a token of my user on Github but encrypted
 - the deploy process will try to push the eplaunch.tar.gz file, if it exists
 - this will only occur on tags (releases) and only if this is the test environment: package